### DISCUSSION TITLE
Serverless Functions limit when deploying SSG/getStatixProps to hobby plan.

### DISCUSSION THREAD

**Discussion by leighs-hammer** [vercel/vercel#4028] [General] [answered]
Hi all, does any one know how the limit of serverless functions is calculated on now.sh deploys?

I was testing out getstaticprops and hit a limit on a 100% ssg build ( hobby plan my personal one )

Any one else seen that? It's related to the number of static pages I was loading. Removing a few ( only 8 articles n total), seemed to not hit the limit.

Is that to be expected personal plans and experimental can load more than 12 articles/pages. When it does build the functions show as 0 so wandering how it is calculated :)

**Comment by Vadorequest** [comment]
@leighs-hammer Mentioned this at https://github.com/zeit/now/discussions/4029

**Comment by mcsdevv** [comment, marked as answer]
Hey @leighs-hammer, this wasn't working quite right previously but should be fixed now. Moving forward, Next.js Static Site Generation and Next.js Preview Serverless Functions are excluded from the count for "Maximum Functions per Deployment".
